scripts/fonts.json

```diff
@@ -31,13 +31,7 @@
     "Noto Sans Telugu": ["regular", "700"],
     "Noto Sans Thai": ["regular", "700"]
   },
-  "bundle-font-stacks": {
-    "Noto Sans HK": ["regular", "700"],
-    "Noto Sans JP": ["regular", "700"],
-    "Noto Sans KR": ["regular", "700"],
-    "Noto Sans SC": ["regular", "700"],
-    "Noto Sans TC": ["regular", "700"]
-  },
```

Comment:
I assume this was removed for performance reasons. We should ticket out any regression in rendering CJK.

Reply:
Yes, and they're not actually used yet.

Reply:
Ticketed in #872